platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -10,6 +10,8 @@ supervisorctl start syncd
 
 # If this platform has an initialization file for the Broadcom LED microprocessor, load it
 if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
+    # Sleep a bit to allow syncd to create the socket for bcmcmd to connect to
+    sleep 5
```

Comment:
Thanks, Pavel. This is intended to be a temporary solution; we will eventually load this file in SAI instead of in this script. However, I agree your solution is more robust, even for this temporary change. Resolved.